Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,199 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace BootCamp.Chapter
 {
     public static class BalanceStats
     {
         /// <summary>
         /// Return name and balance(current) of person who had the biggest historic balance.
-        /// </summary>
+        /// </summary>       
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (IsNullOrEmpty(peopleAndBalances)) return "N/A.";
```

Comment:
"N/A" should be moved to a constant, because now it needs to change what is returned for invalid value- we will have to change multiple places.